instrumentation/opentelemetry-instrumentation-fastapi/tests/test_fastapi_instrumentation.py

```diff
@@ -1839,3 +1839,50 @@ def test_custom_header_not_present_in_non_recording_span(self):
         self.assertEqual(200, resp.status_code)
         span_list = self.memory_exporter.get_finished_spans()
         self.assertEqual(len(span_list), 0)
+
+
+class TestFastAPIRenderPathParameters(TestBaseFastAPI):
```

Comment:
this can't be a new test in any of other test base class? For example, TestWrappedApplication

Reply:
Sorry, this is my first time working with OpenTelemetry and this project, so I'm not very familiar with it. I have moved the test code to `test_fastapi_instrumentation_wrapped.py`. Could you please take another look and see if there are any other issues?

Reply:
No worries. I mean under an existent test class instead of creating a new one

Reply:
I understand what you mean, I have edited it.